### PR TITLE
fix(DEV-11218): improve error handling with unstable response 🕺

### PR DESCRIPTION
**Pull request by radist2s**
Replaced unstable error handling like

```ts
toast.error(error.body.error.message || error.message);
```

with the conventional call of `getLegacyAPIErrorMessage(error)`

* `getLegacyAPIErrorMessage(...)` has been renamed from `getMessageInError(...)`
* `getAPIErrorMessage(...)` has been moved to `core/utils` folder

### CODE REVIEW

File: packages/sdk-react/src/core/utils/getLegacyAPIErrorMessage.test.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { getLegacyAPIErrorMessage } from '@/core/utils/getLegacyAPIErrorMessage';
+
+describe('getMessageInError', () => {
+  it('should return the message from an error object', () => {
```
Comment (nekedos) [inline packages/sdk-react/src/core/utils/getLegacyAPIErrorMessage.test.ts:4]: Can I ask to add more understandable test names to avoid duplicates (line 16)?

Comment (radist2s) [reply packages/sdk-react/src/core/utils/getLegacyAPIErrorMessage.test.ts]: Thank you, fixed, and improved others.

**Review by nekedos — APPROVED** [review]
